core/templates/App.ts

```diff
@@ -15,7 +15,8 @@
 /**
  * @fileoverview Initialization and basic configuration for the Oppia module.
  */
-
+require('@angular/material/prebuilt-themes/deeppurple-amber.css');
```

Comment:
Do we had this theme earlier? Do we have any alternatives?

Reply:
No, we didn't. The reason to import this theme was to get material css. There are four prebuilt themes but I suggest we don't go for them, instead, we should do this: https://material.angular.io/guide/theming-your-components#theming-your-custom-component-with-angular-materials-theming-system.

Reply:
But that need Sass which we don't use as of now, right? ANy other alternatives. Also, can you explain the impact of choosing a theme here (with some UI examples).

Reply:
There is no immediate impact, however, if we use the material directives, this CSS may override the oppia.css. In that case, we should be make our CSS rules more specific.

Reply:
> ANy other alternatives.

This is the best material design UI library for angular2+ (Made by the angular team for angular 2+). I would say first-party over third party libraries any day.

Reply:
Also, please note that this change has been brought with the intention of easing migration of angularjs material components as this library is very similar to angularjs material.